docs/low-code-sql/data-tests/data-tests.md

```diff
@@ -9,15 +9,22 @@ tags:
   - sql
 ---
 
-You can use data tests to ensure that your business data is generated reliably over time. As a data engineer, data analyst, or business user, you can run data tests so that you don’t have to manually check every Dataset every time you run a Job or model. The data test checks the validity of the SQL in your project by querying the single table result generated from the transformation steps shown.
+You can use data tests to ensure that your business data is generated reliably over time. As a data engineer, data analyst, or business user, you can run data tests so that you don’t have to manually check every Dataset every time you run a Job or model. The data test checks the validity of the SQL in your project.
+
+For example, find below a **Test** named `ref_int_orders_customers` which checks the validity of the SQL in the `HelloWorld_SQL` Project. In particular, the referential integrity check for orders and customers asserts that every `customer_id` entry in the `orders` table is present in the `customers` table.
 
 ![Project test canvas](img/project-test-canvas.png)
+This test starts with several **(1) models** from the `HelloWorld_SQL` Project, combines their data with a series of **(2) transformation** steps, and the resulting table is fed into the **(3) Data Test** Gem.
+
+If there are `customer_id` entries in the `orders` table that are not present in the `customers` table, then the `ref_int_orders_customers` test will fail.
+
+Indeed any series of transformations can be tested with a Data Test Gem.
 
 ## What you'll need to know
 
-Data tests use [dbt](https://docs.getdbt.com/docs/build/data-tests) for the underlying test execution, but you don’t need to know dbt or how to write your own tests. Prophecy simplifies the test definitions that are normally defined in `.sql` and `.yaml` files.
```

Comment:
I suggest moving this to the end of this doc.

Reply:
I left this here since it fits in this section for what users will need / or not need to know. The below section works better to help users move onto the next topic. A Deep Dive section is confusing to me what users might expect to find there.